odl/tomo/backends/astra_setup.py

```diff
@@ -74,6 +74,70 @@
            'astra_parallel_3d_geom_to_vec')
 
 
+# Dictionary specifying which ASTRA versions support which feature. The dict
+# values can be given as exact version '==1.7' or '1.7', as range '1.7-1.7.4',
+# as inequality '>=1.7' or as comma-separated composition of such (the items
+# will be OR-ed together), e.g. '<=1.7, >=1.7.2'.
+ASTRA_FEATURES = {'anisotropic_voxels_2d': None,
+                  'anisotropic_voxels_3d': '>=1.8',
+                  }
+
+
+def astra_supports(feature):
```

Comment:
I very much doubt there is something like this in ASTRA. And although this is a bit of code, I think it makes it easier to specify features of versions, compared to the alternative (that I started with) of hard-coding variables like
```
ASTRA_SUPPORTS_ANISO_VOXELS_3D = (
    parse_version(ASTRA_VERSION) >= parse_version('1.8'))
```
which becomes messy with broken intermediate versions like 1.7.1 which screwed up axis-aligned `origin_to_det`, for instance.

I'll see if I can move it to a util module, however it should still be under `backends/`.